.ghci800

```diff
@@ -40,6 +40,9 @@
 :set -Wunused-foralls
 :set -Wwarnings-deprecations
 :set -Wwrong-do-bind
+:set -Wunused-imports
+:set -Wunused-top-binds
```

Comment:
I'm getting warnings during compilation now about unused bindings. Did you mean to add this flag after removing the top-level bindings?

Reply:
Ooops, I thought if I can remove those top-level bindings quickly enough, then no one would see those warnings. 

Check #3922. :-)